BrainPortal/app/controllers/carmin_controller.rb

```diff
@@ -128,7 +128,7 @@ def executions #:nodoc:
     limit      = params[:limit].presence
 
     if group_name
-      group = current_user.available_groups.where('groups.name' => group_name).first
+      group = current_user.viewable_groups.where('groups.name' => group_name).first
```

Comment:
Why viewable_groups here, and assignable_groups later ? 
If I get it correctly you use the viewable_groups for GET and assignable_groups when you POST.

Reply:
Yeah. I guess this was so a carmin user could see tasks that are public or in group 'everyone', but really it's not a circumstance I can imagine occurring frequently, so I'll change it back to assignable_groups()